libs/service-portal/core/src/components/Modal/Modal.tsx

```
@@ -24,12 +43,39 @@ export const Modal: FC<React.PropsWithChildren<Props>> = ({
+  buttons,
   initialVisibility = true,
   skeleton,
+  iconAlt,
+  iconSrc,
 }) => {
+  const [closing, setClosing] = useState(false)
+  const [startClosing, setStartClosing] = useState(false)
+
+  useEffect(() => {
+    if (closing) {
+      onCloseModal && onCloseModal()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)